app/forms/support_interface/editable_until_form.rb

```diff
@@ -0,0 +1,54 @@
+module SupportInterface
+  class EditableUntilForm
+    include ActiveModel::Model
+    attr_accessor :application_form, :audit_comment, :audit_comment_description
+    attr_writer :sections, :editable_until
+
+    validates :audit_comment, presence: true
+    validates_with ZendeskUrlValidator
```

Comment:
Should it validate at least one `editable_section` checked?

Reply:
The requirement is if we left all unchecked and add a ticket we return the application to clean slate (the normal editable / non editable section config). Sorry, I should had put into the PR description.